testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveCoercionOnUnpartitionedTable.java

```diff
@@ -43,37 +43,40 @@ private static HiveTableDefinition.HiveTableDefinitionBuilder tableDefinitionBui
         return HiveTableDefinition.builder(tableName)
                 // all nested primitive coercions and adding/removing trailing nested fields are covered across row_to_row, list_to_list, and map_to_map
                 .setCreateTableDDLTemplate("""
-                        CREATE TABLE %NAME%(
-                            row_to_row                         STRUCT<keep: STRING, ti2si: TINYINT, si2int: SMALLINT, int2bi: INT, bi2vc: BIGINT, lower2uppercase: BIGINT>,
-                            list_to_list                       ARRAY<STRUCT<ti2int: TINYINT, si2bi: SMALLINT, bi2vc: BIGINT, remove: STRING>>,
-                            map_to_map                         MAP<TINYINT, STRUCT<ti2bi: TINYINT, int2bi: INT, float2double: FLOAT>>,
-                            tinyint_to_smallint                TINYINT,
-                            tinyint_to_int                     TINYINT,
-                            tinyint_to_bigint                  TINYINT,
-                            smallint_to_int                    SMALLINT,
-                            smallint_to_bigint                 SMALLINT,
-                            int_to_bigint                      INT,
-                            bigint_to_varchar                  BIGINT,
-                            float_to_double                    FLOAT,
-                            double_to_float                    DOUBLE,
-                            shortdecimal_to_shortdecimal       DECIMAL(10,2),
-                            shortdecimal_to_longdecimal        DECIMAL(10,2),
-                            longdecimal_to_shortdecimal        DECIMAL(20,12),
-                            longdecimal_to_longdecimal         DECIMAL(20,12),
-                            float_to_decimal                   FLOAT,
-                            double_to_decimal                  DOUBLE,
-                            decimal_to_float                   DECIMAL(10,5),
-                            decimal_to_double                  DECIMAL(10,5),
-                            short_decimal_to_varchar           DECIMAL(10,5),
-                            long_decimal_to_varchar            DECIMAL(20,12),
-                            short_decimal_to_bounded_varchar   DECIMAL(10,5),
-                            long_decimal_to_bounded_varchar    DECIMAL(20,12),
-                            varchar_to_bigger_varchar          VARCHAR(3),
-                            varchar_to_smaller_varchar         VARCHAR(3),
-                            char_to_bigger_char                CHAR(3),
-                            char_to_smaller_char               CHAR(3),
-                            id                                 BIGINT)
-                       STORED AS\s""" + fileFormat);
+                         CREATE TABLE %NAME%(
```

Comment:
This is looks like unintended format change. If that was the idea, please put it into a separate commit

Reply:
It was IntelliJ formatting messed this change. Thanks for pointing this one.